portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -73,6 +73,92 @@ definition {
 		}
 	}
 
+	@description = "LPS-119966 Given the Content Dashboard when a document is listed in the assets list then the following metadata are shown: Title, Author (Owner), Subtype, Site or Asset Library, Status, Category 1, Category 2, Modified Date"
+	@priority = "3"
+	test AssetListContent {
+		property portal.acceptance = "true";
+
+		task ("Create Category 1 and 2") {
+			JSONCategory.addVocabulary(
+				groupName = "Test Site Name",
+				title = "Vocabulary Name");
+
+			var categoryId1 = JSONCategory.addCategory(
+				groupName = "Test Site Name",
+				title = "Category 1",
+				vocabularyName = "Vocabulary Name");
+
+			var categoryId2 = JSONCategory.addCategory(
+				groupName = "Test Site Name",
+				title = "Category 2",
+				vocabularyName = "Vocabulary Name");
+		}
+		task ("Create WC via JSON and select categories") {
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "WC WebContent Title");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContentNavigator.gotoEditCP(webContentTitle = "WC WebContent Title");
+
+			WebContent.selectCategory(
+				categoryName = "Category 1",
+				vocabularyName = "Vocabulary Name");
+
+			WebContent.selectCategory(
+				categoryName = "Category 2",
+				vocabularyName = "Vocabulary Name");
+
+			PortletEntry.publish();
+		}
```

Comment:
You can add a web content with multiple categories:

WebContent.addWithMultipleCategories(
			categoryNameList = "Category Name 1,Category Name 2,Category Name 3,Category Name 4,Category Name 5",
			vocabularyName = "Vocabulary Name",
			webContentContent = "WC Content Body",
			webContentTitle = "WC Content Title");